src/main/java/seedu/address/logic/commands/UntagCommand.java

```diff
@@ -0,0 +1,195 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.lecture.Lecture;
+import seedu.address.model.lecture.LectureName;
+import seedu.address.model.lecture.ReadOnlyLecture;
+import seedu.address.model.module.Module;
+import seedu.address.model.module.ModuleCode;
+import seedu.address.model.module.ReadOnlyModule;
+import seedu.address.model.tag.Tag;
+import seedu.address.model.video.Video;
+import seedu.address.model.video.VideoName;
+
+/**
+ * Tag a video, a lecture, or a module.
+ */
+
+public class UntagCommand extends Command {
+    public static final String COMMAND_WORD = "untag";
+
+    //TODO: MODIFY THIS
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Untag a specified video, module, or lecture ";
+    //TODO: MODIFY THIS
+    public static final String MESSAGE_SUCCESS = "Item untagged";
+    public static final String MESSAGE_TAG_NOT_FOUND = "Tag doesn't exist";
+    public static final String MESSAGE_MODULE_NOT_FOUND = "Module doesn't exist in Le Tracker";
+    public static final String MESSAGE_LECTURE_NOT_FOUND = "Lecture doesn't exist in this module";
+    public static final String MESSAGE_VIDEO_NOT_FOUND = "Video doesn't exist in this lecture";
```

Comment:
Can possibly use the Messages.java instead? If any variable is not available there, you can move them it there